src/components/variant/Variant.tsx

```diff
@@ -72,17 +77,17 @@ export class Variant extends React.Component<Props, State> {
     return displayPriority[a] > displayPriority[b] ? 1 : -1;
   }
 
-  private orderByPriority() {
+  private orderByPriority(statusCount: {}) {
```

Comment:
Since you use the statusCount type declaration in 2 places now, can you create a private type for it in this file and declare this param as well as this.state.statusCount as that type?

Reply:
yup, declared a new type: 
interface StatusCount { [id: string]: number };